internal/proto/volumegroup.proto

```diff
@@ -0,0 +1,182 @@
+syntax = "proto3";
+package proto;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/proto";
+
+// Controller holds the RPC methods for volumeGroup and all the methods it
+// exposes should be idempotent.
+service Controller {
+  // CreateVolumeGroup RPC call to create a volume group.
+  rpc CreateVolumeGroup(CreateVolumeGroupRequest)
+      returns (CreateVolumeGroupResponse) {}
+  // ModifyVolumeGroupMembership RPC call to modify a volume group.
+  rpc ModifyVolumeGroupMembership(ModifyVolumeGroupMembershipRequest)
+      returns (ModifyVolumeGroupMembershipResponse) {}
+  // DeleteVolumeGroup RPC call to delete a volume group.
+  rpc DeleteVolumeGroup(DeleteVolumeGroupRequest)
+      returns (DeleteVolumeGroupResponse) {}
+  // ListVolumeGroups RPC call to list volume groups.
```

Comment:
Lots of things commented out, just remove it all.

This .proto file is for the communication between controller and sidecar, it would (usually) be a reduced/simplified version of the CSI-Addons specification. You can use Kubernetes objects/references here, PVs instead of CSI volume_ids and so on.